src/server/metrics/github-users.js

```diff
@@ -0,0 +1,12 @@
+import promClient from 'prom-client';
+
+import { GitHubUser } from '../db/models/github-user';
+
+const gitHubUsersTotal = new promClient.Gauge(
```

Comment:
Not sure if it is worth the effort (because I guess promClient would need to be mocked), but maybe it would be worth having at least simple unit test that verifies that we expose the metrics?